livereveal/main.js

```diff
@@ -32,7 +30,15 @@ var config = new configmod.ConfigWithDefaults(config_section, {
     slideNumber: true,
     start_slideshow_at: 'beginning',
     scroll: false,
-});
+};
+
+if(Jupyter.notebook.metadata.livereveal != undefined){
+    default_config = $.extend(true, default_config, Jupyter.notebook.metadata.livereveal);
+} else {
+    config_section.load();
```

Comment:
This should be before the `if`, otherwise, if you trigger the condition, it is never executed... why you put it here?